src/update_notifier.rs

```diff
@@ -91,9 +91,16 @@ impl WapmUpdate {
                 let new_version = last_check.version.to_owned();
                 let old_version = util::get_latest_runtime_version()?;
 
-                // If we are in the same version
-                if old_version == new_version {
-                    return Ok(());
+                if let Some(b) = util::compare_versions(&old_version, &new_version) {
```

Comment:
This might be a bit simpler?

```suggestion
                if util::compare_versions(&old_version, &new_version).unwrap_or(false) {
```
(removing the `if b` line after)